packages/@react-spectrum/button/stories/ActionButton.stories.tsx

```diff
@@ -24,11 +25,45 @@ storiesOf('Button/ActionButton', module)
   )
   .add(
     'icon',
-    () => render({icon: <Add />})
+    () => (
```

Comment:
is there a reason not to just pass the icon this way?
```render({children: <Add />})```

Reply:
Yeah I suppose I could use a fragment to pass both the icon and text.